### PR TITLE
Bonnie 1132 automate 918

### PR DESCRIPTION
**Pull request by lukoktonos**
These changes replace a JIRA test that was testing direct reference code usage in the Patient Builder View.

NOTE: JIRA ticket mentions to test loading of direct reference code, which is already tested as part of "direct reference codes update properly during rebuild elm" in cql_test.rb

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1132
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 

Branch | Back End Coverage | Front End Coverage
-- | -- | --
master | 1689 / 1917 LOC (88.11%) covered. | Statements   : 57.06% ( 6833/11975 ) <br> Branches     : 40% ( 1789/4473 ) <br> Functions    : 47.62% ( 1129/2371 ) <br> Lines        : 54.81% ( 6116/11159 )
bonnie-1132-automate-918 | 1689 / 1917 LOC (88.11%) covered. | Statements   : 57.15% ( 6844/11975 ) <br> Branches     : 40.17% ( 1797/4473 ) <br> Functions    : 47.7% ( 1131/2371 ) <br>Lines        : 54.91% ( 6127/11159 )

- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @c-monkey 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: spec/javascripts/production_tests/production_patient_builder_spec.js.coffee [hunk 1]
```diff
@@ -90,12 +90,15 @@ describe 'Production_PatientBuilderView', ->
       bonnie.valueSetsByOid = @universalValueSetsByOid
       bonnie.measures = @bonnie_measures_old
 
-    describe 'Patient "Numer PASS"', ->
```
Comment (c-monkey) [inline spec/javascripts/production_tests/production_patient_builder_spec.js.coffee]: This change seems arbitrary and now the quotes don't match.

File: spec/javascripts/production_tests/production_patient_builder_spec.js.coffee [hunk 2]
```diff
@@ -106,3 +109,48 @@ describe 'Production_PatientBuilderView', ->
...
+
+    afterEach ->
+      bonnie.valueSetsByOid = @universalValueSetsByOid
+      bonnie.measures = @bonnie_measures_old
+
+    describe 'Patient Direct Reference Code Element', ->
+      beforeEach ->
+        @patient = @patients.findWhere(first: 'Element', last: 'Direct Reference Code')
+        @patientBuilderView = new Thorax.Views.PatientBuilder(model: @patient, measure: @measure, patients: @patients, measures: bonnie.measures, inPatientDashboard: false)
+        @selectCriteriaItemView = new Thorax.Views.SelectCriteriaItemView(model: @measure.get('source_data_criteria').models[18])
```
Comment (c-monkey) [inline spec/javascripts/production_tests/production_patient_builder_spec.js.coffee]: I don't know how likely the index 18 is to change. I suggest we move the below assertion to here (i.e. assert `@measure.get('source_data_criteria').models[18].get('description')` matches) - this would be more explicit than checking the model of the SelectCriteriaItemView. This is probably what you intended but the distance from the `[18]` to the assertion is 7 lines. If the assertion arises someday, it might not be as obvious why it is failing.

**Review by c-monkey — CHANGES_REQUESTED** [review]
trivial changes requested, looks good, tests pass, DRC uses properly tested replacing manual test

File: spec/javascripts/production_tests/production_patient_builder_spec.js.coffee (hunk 1 above)
Comment (lukoktonos) [reply spec/javascripts/production_tests/production_patient_builder_spec.js.coffee]: For some reason my brain didn't process the other `"` so I thought I was actually fixing a mismatched quote.  Good catch.

**Review by mayerm94 — APPROVED** [review]
Test matches purpose of manual tests. Changes look good.

**Review by c-monkey — APPROVED** [review]
